python/example_code/glue/glue_wrapper.py

```diff
@@ -0,0 +1,319 @@
+        """
+        try:
+            self.glue_client.create_crawler(
+                Name=name,
+                Role=role_arn,
+                DatabaseName=db_name,
+                TablePrefix=db_prefix,
+                Targets={'S3Targets': [{'Path': s3_target}]})
+        except ClientError as err:
+            logger.error(
+                "Couldn't create crawler. Here's why: %s: %s",
+                err.response['Error']['Code'], err.response['Error']['Message'])
+            raise
+    # snippet-end:[python.example_code.glue.CreateCrawler]
+
+    # snippet-start:[python.example_code.glue.StartCrawler]
+    def start_crawler(self, name):
+        """
+        Starts a crawler. The crawler crawls its configured target and creates
+        metadata that describes the data it finds in the target data source.
+
+        :param name: The name of the crawler to start.
+        """
+        try:
+            self.glue_client.start_crawler(Name=name)
+        except ClientError as err:
+            logger.error(
+                "Couldn't start crawler %s. Here's why: %s: %s", name,
+                err.response['Error']['Code'], err.response['Error']['Message'])
+            raise
+    # snippet-end:[python.example_code.glue.StartCrawler]
+
+    # snippet-start:[python.example_code.glue.GetDatabase]
+    def get_database(self, name):
+        """
+        Gets information about a database in your Data Catalog.
+
+        :param name: The name of the database to look up.
+        :return: Information about the database.
+        """
+        try:
+            response = self.glue_client.get_database(Name=name)
+        except ClientError as err:
+            logger.error(
+                "Couldn't get database %s. Here's why: %s: %s", name,
+                err.response['Error']['Code'], err.response['Error']['Message'])
+            raise
+        else:
+            return response['Database']
+    # snippet-end:[python.example_code.glue.GetDatabase]
+
+    # snippet-start:[python.example_code.glue.GetTables]
+    def get_tables(self, db_name):
+        """
+        Gets a list of tables in a Data Catalog database.
+
+        :param db_name: The name of the database to query.
+        :return: The list of tables in the database.
+        """
+        try:
+            response = self.glue_client.get_tables(DatabaseName=db_name)
+        except ClientError as err:
+            logger.error(
+                "Couldn't get tables %s. Here's why: %s: %s", db_name,
+                err.response['Error']['Code'], err.response['Error']['Message'])
+            raise
+        else:
+            return response['TableList']
+    # snippet-end:[python.example_code.glue.GetTables]
+
+    # snippet-start:[python.example_code.glue.CreateJob]
+    def create_job(self, name, description, role_arn, script_location):
+        """
+        Creates a job definition for an extract, transform, load (ETL) job that can
```

Comment:
extract, transform, load (ETL) --> extract, transform, and load (ETL)

Reply:
fixed